### DISCUSSION TITLE
Time Control Quick Create Editor not working on Yetiforce 6.2

### DISCUSSION THREAD

**Discussion by staples1347** [YetiForceCompany/YetiForceCRM#15272] [General]
On the Yetiforce 6.2 demo, Time Control is using a new Quick Create Editor that looks a lot better than the legacy editor.  On a self hosted installation that I upgraded from Yetiforce 5.2 to Yetiforce 6.2, I am still getting the old Time Control interface.  Also, in Standard Modules, when I click on Settings for Time Control and then click "Quick Create Editor", it throws an error: "Call to a member function getFieldInfo() on bool" .  Is anyone else experiencing this problem?

This problem is occurring on our installation of Yetiforce 6.2

**Comment by rskrzypczak** [comment]
Install and use version 6.2 or upgrade your system to 6.2 :)
